docs/virtual_machines/templates.md

```diff
@@ -45,7 +45,7 @@ releases](https://github.com/kubevirt/common-templates/releases)
 
 To install the templates:
 
-    $ export VERSION="v0.3.1"
+    $ export VERSION="v0.13.1"
```

Comment:
We need to `curl` latest release.  We keep advertising old versions then having to come back and adjust these versions

Please use ...
`VERSION=$(curl -s https://api.github.com/repos/kubevirt/common-templates/releases | grep tag_name | grep -v -- '-rc' | head -1 | awk -F': ' '{print $2}' | sed 's/,//' | xargs)`